CHANGELOG.md

```diff
@@ -83,6 +83,7 @@
 -   Tab bar text now aligned and centered correctly - luc
 -   Adds iPad support for Auctions view layout - luc
 -   Adds <LotsByArtistsYouFollow /> component to Sales - chris
+-   Refactors <LotsByArtistsYouFollow /> by consolidating QueryRenderers - chris
```

Comment:
FYI Changes that haven’t gone out in a release before don’t need to be mentioned in the CHANGELOG.

/cc @l2succes

Reply:
Ah, this was getting flagged by Danger; cc @orta

Reply:
Yeah but that’s more because this overall work is split across 2 PRs :)

Reply:
@orta - how do we get around this Danger `FAIL` issue in circumstances like this? "Skip Changelog"?